lib/query/analytic.js

```diff
@@ -18,7 +19,7 @@ class Analytic {
     this.grouping = 'columns';
   }
 
-  partitionBy(column) {
+  partitionBy(column, direction) {
```

Comment:
This needs to be documented. Could you create a PR for https://github.com/knex/documentation?

Reply:
https://github.com/knex/documentation/pull/333